plugin/core/protocol.py

```diff
@@ -384,9 +399,14 @@ class CodeActionKind(StrEnum):
     They should not suppress errors or perform unsafe fixes such as generating new types or classes.
 
     @since 3.15.0 """
+    Notebook = 'notebook'
+    """ Base kind for all code actions applying to the entire notebook's scope. CodeActionKinds using
+    this should always begin with `notebook.`
+
+    @since 3.18.0 """
 
 
-class TraceValues(StrEnum):
+class TraceValue(Enum):
```

Comment:
Why did this change from `StrEnum` to `Enum`?

Reply:
It was always Enum, but was changed to StrEnum in this PR https://github.com/sublimelsp/LSP/pull/2446/files#diff-049653877a86e4c220cd537d174655150e80b4ce0077d48742cc72b4654e2112R388

In the metaModel.json schema it is typed as Enum.
I'll leave it as Enum, because `TraceValue` is not used anywhere in code yet.

Reply:
> In the metaModel.json schema it is typed as Enum.

In metaModel.json there is no concept of StrEnum or IntEnum so that's not really an argument. :)

I think the answer to my question is that the name of it has changed but the https://github.com/sublimelsp/lsp-python-types/blob/f48ed1acee6040d64fc2d69df4eee77667008fcf/generate.py#L26 wasn't updated.

I think `StrEnum` would be correct here but if we are not using it then I guess it doesn't matter.

Reply:
you are correct, thanks for the lookup

Reply:
I've pushed a fixed https://github.com/sublimelsp/lsp-python-types/commit/0da4d7c4d7543404d9a322f487280990d04980af